src/Logger.js

```diff
@@ -5,12 +5,23 @@ const Config = require('./Config.js');
 const Logger = bunyan.createLogger(Config.loggerParams());
 
 function LogError(err, context) {
+    Log(err, context, "error");
+}
+
+function LogExceptionMessage(err, context) {
```

Comment:
I would call this LogException(), especially since its sister function is _not_ called LogErrorMessage().

Reply:
Done.